index.ts

```diff
@@ -1,18 +1,27 @@
 import { state, nodes } from "membrane";
 
-// Click `Invoke ►` above the `run` fn signature, then see Logs (bottom) and check your email
+// Click `Invoke ►` (below this comment) to send yourself a welcome email!
 export async function run() {
-  console.log("Hello World");
+  console.log("Sending email...");
 
-  // `nodes` is how you connect to your other programs
-  await nodes.email.send({ subject: "Welcome to Membrane", body: "👋" });
+  // `nodes` holds connections to other programs (see `email` on the right sidebar)
+  await nodes.email.send({ subject: "Hello World", body: "👋 Welcome to Membrane!" });
 }
 
-// Click `Open Preview` (above fn signature), then check Logs and state (bottom left)
+// Click `Open Preview` (below this comment), then check Logs and state (right sidebar)
 export async function endpoint({ method, path, body, headers, query }) {
-  // `state` is like a little db for your program
-  if (!state.hits) state.hits = 0;
-  state.hits++;
-
-  return `Visitors: ${state.hits}`;
+  // This program is durable! `state` is like a little database for your program
+  // but you can store any JavaScript object.
+  if (!state.hits) {
+    state.hits = 0;
+  }
+  
+  return `Visits: <p>${state.hits}`;
```

Comment:
I think we still need to increment hits?
```suggestion
  return `Visits: <p>${++state.hits}`;
```